src/lithopscloud/modules/utils.py

```diff
@@ -240,9 +237,8 @@ def _is_valid_input_path(path):
         return True
 
     def _is_valid_output_path(path):
-        dir_file = path.rsplit('/', 1)
-        prefix_directory = dir_file[0]
-        if len(dir_file) == 1 or os.path.isdir(prefix_directory):
+        prefix_directory = path.rsplit('/', 1)[0]
```

Comment:
please, return the previous implementation.
i fixed it previously to support local filename without specifying whole path `-o myfile.yaml`